assets/pages/subscriptions-landing/subscriptionsLanding.jsx

```diff
@@ -19,7 +19,6 @@ import PaperSubscriptionsContainer from 'components/paperSubscriptions/paperSubs
 import { init as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
 
-import { sendTrackingEventsOnClick } from 'helpers/subscriptions';
```

Comment:
I think this method might be unused in the project now in which case we could remove it as well

Reply:
You're right, `yarn validate` spotted that too 😃Should have gone now.

Reply:
@rupertbates did you mean delete the implementation from `subscriptions.js` as well?

Reply:
If it's not used I should probably bin it from there too

Reply:
We can pull it back if needed later

Reply:
Having said that, I imagine we will almost definitely do more of this kind of test tracking in the near future? Maybe it's better to leave it in?
#techdebt